buildbot/check.py

```diff
@@ -35,7 +73,9 @@ def main():
                         help="builder directory, which is the directory contains source and build directories")
     parser.add_argument("-s", "--src-dir", metavar="SRC_DIR", help="source directory")
     parser.add_argument("-o", "--obj-dir", metavar="OBJ_DIR", required=True, help="build directory")
-    parser.add_argument("-t", "--test-suite", metavar="TEST_SUITE", required=True, help="check-xxx target")
+    parser.add_argument("-t", "--test-suite", metavar="LIT_TEST_SUITE", required=True, help="check-xxx target")
```

Comment:
You updated `metavar`, but left usages of `test_suite` field in the code - I guess they also need to be updated (line 18 in this file)

Reply:
Good catch. Now need to figure out why my local testing worked as expected before fixing this.

Reply:
From [Python docs metavar](https://docs.python.org/3/library/argparse.html#metavar):
> Note that metavar only changes the displayed name - the name of the attribute on the parse_args() object is still determined by the dest value

Though this might be confusing - should I change the metavar value back to `TEST_SUITE`?